Integrations/EclecticIQ/EclecticIQ.py

```diff
@@ -0,0 +1,640 @@
+''' IMPORTS '''
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import json
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+USERNAME = demisto.params().get('credentials').get('identifier')
+PASSWORD = demisto.params().get('credentials').get('password')
+URL = demisto.params()['url']
+SERVER = URL[:-1] if (URL and URL.endswith('/')) else URL
+USE_SSL = not demisto.params().get('insecure', False)
+HEADERS = {}
+IP_THRESHOLD = demisto.params().get('ip_threshold').lower()
+URL_THRESHOLD = demisto.params().get('url_threshold').lower()
+FILE_THRESHOLD = demisto.params().get('file_threshold').lower()
+EMAIL_THRESHOLD = demisto.params().get('email_threshold').lower()
+DOMAIN_THRESHOLD = demisto.params().get('domain_threshold').lower()
+
+if not demisto.params().get('proxy'):
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
+    PROXIES = {
+        'http': None,
+        'https': None
+    }
+else:
+    PROXIES = {
+        'http': os.environ['http_proxy'] or os.environ['HTTP_PROXY'],
+        'https': os.environ['https_proxy'] or os.environ['HTTPS_PROXY']
+    }
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, url_suffix, headers=HEADERS, cmd_json=None):
+    res = requests.request(
+        method,
+        SERVER + url_suffix,
+        headers=headers,
+        json=cmd_json,
+        proxies=PROXIES,
+        verify=USE_SSL
+    )
+    if res.status_code not in {200}:
+        if res.status_code == 405:
```

Comment:
shouldn't this be 404?
405 seems to be more relevant to bug in the code and not the parameter.

Reply:
well, i didn't choose the status codes.. but yea its 405

Reply:
just wanted to make sure it's not a typo